CHANGELOG.md

```diff
@@ -18,6 +18,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Bugfix: Virtual Cameras were not updating in Edit mode when Brain's BlendUpdateMode was FixedUpdate.
 - Bugfix: Lens mode override was not working correctly in all cases.
 - Collider2D inspector: added warning when collider is of the wrong type.
+- Added ability to directly set the active blend in CinemachineBrain.
```

Comment:
Was this line just missing, or did it creep here by mistake?

Reply:
I changed the merge strategy for the CHANGELOG to union. That works well for PRs, but it shits all over backports. I cleaned it up in b0533c2 (see below).